server/src/domain/media/media.util.ts

```diff
@@ -50,7 +50,7 @@ class BaseConfig implements VideoCodecSWConfig {
     if (this.shouldToneMap(stream)) {
       options.push(...this.getToneMapping());
     }
-    options.push('format=yuv420p');
+    options.push(`format=yuv420p`);
```

Comment:
?

Reply:
Ah, this is a holdover from my FFmpeg attempt.